apps/web/pages/bookings/[status].tsx

```diff
@@ -58,7 +58,19 @@ export default function Bookings() {
     }
     return { recurringCount };
   };
-
+  const shownBookings: Record<string, boolean> = {};
+  const filterBookings = (booking: BookingOutput) => {
+    if (status === "recurring" || status === "cancelled") {
```

Comment:
We want to group recurring event instances into just one for cancelled and recurring tabs